downloads.html

```diff
@@ -9,48 +9,75 @@
     <p>
     You can verify your download by following these <a href="http://www.apache.org/info/verification.html">procedures</a> and using these <a href="http://kafka.apache.org/KEYS">KEYS</a>.
     </p>
+        <h3>1.0.0</h3>
 
-			<h3>1.0.0</h3>
-			<ul>
-					<li>
-							Released November 1, 2017
-					</li>
-					<li>
-							<a href="https://dist.apache.org/repos/dist/release/kafka/1.0.0/RELEASE_NOTES.html">Release Notes</a>
-					</li>
-					<li>
-							Source download: <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/1.0.0/kafka-1.0.0-src.tgz">kafka-1.0.0-src.tgz</a> (<a href="https://dist.apache.org/repos/dist/release/kafka/1.0.0/kafka-1.0.0-src.tgz.asc">asc</a>, <a href="https://dist.apache.org/repos/dist/release/kafka/1.0.0/kafka-1.0.0-src.tgz.sha512">sha512</a>)
-					</li>
-					<li>
-							Binary downloads:
-							<ul>
-									<li>Scala 2.11 &nbsp;- <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/1.0.0/kafka_2.11-1.0.0.tgz">kafka_2.11-1.0.0.tgz</a> (<a href="https://dist.apache.org/repos/dist/release/kafka/1.0.0/kafka_2.11-1.0.0.tgz.asc">asc</a>, <a href="https://dist.apache.org/repos/dist/release/kafka/1.0.0/kafka_2.11-1.0.0.tgz.sha512">sha512</a>)</li>
-									<li>Scala 2.12 &nbsp;- <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/1.0.0/kafka_2.12-1.0.0.tgz">kafka_2.12-1.0.0.tgz</a> (<a href="https://dist.apache.org/repos/dist/release/kafka/1.0.0/kafka_2.12-1.0.0.tgz.asc">asc</a>, <a href="https://dist.apache.org/repos/dist/release/kafka/1.0.0/kafka_2.12-1.0.0.tgz.sha512">sha512</a>)</li>
-							</ul>
-							We build for multiple versions of Scala. This only matters if you are using Scala and you want a version built for the same Scala version you use. Otherwise any version should work (2.11 is recommended).
-					</li>
-			</ul>
+        <ul>
+            <li>
+                Released November 1, 2017
+            </li>
+            <li>
+                Source download: <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/1.0.0/kafka-1.0.0-src.tgz">kafka-1.0.0-src.tgz</a> (<a href="https://dist.apache.org/repos/dist/release/kafka/1.0.0/kafka-1.0.0-src.tgz.asc">asc</a>, <a href="https://dist.apache.org/repos/dist/release/kafka/1.0.0/kafka-1.0.0-src.tgz.md5">md5</a>)
```

Comment:
I was linking to the sha512 file since md5 is deprecated these days.

Reply:
ack.